packages/block/test/eip4844block.spec.ts

```diff
@@ -133,16 +133,17 @@ tape('data gas tests', async (t) => {
     t.equal(excessDataGas, 0n, 'excess data gas should be 0 for small parent header data gas')
     t.equal(dataGasPrice, 1n, 'data gas price should be 1n when low or no excess data gas')
     const highGasHeader = BlockHeader.fromHeaderData(
-      { number: 1, excessDataGas: 4194304, dataGasUsed: BigInt(4) * dataGasPerBlob },
+      { number: 1, excessDataGas: 6291456, dataGasUsed: BigInt(6) * dataGasPerBlob },
```

Comment:
Could be worth declaring constants BLOB_PER_BLOCK_TARGET and BLOB_PER_BLOCK_MAX ? Imo it would make it clearer to follow what's going on + easier to play with parameters & update when necessary.

Reply:
BLOB_PER_BLOCK_TARGET and BLOB_PER_BLOCK_MAX are already calculated from the hardfork params (which is same)

tbh i don't see the need unless you feel strongly

Reply:
Ah fair enough then, I agree, no need to add that.